fault/verilator_target.py

```diff
@@ -252,6 +277,7 @@ def make_poke(self, i, action):
                                    i in range(value.file.chunk_size))
                 value = f"({value}) & 0x{mask}"
             value = self.process_value(action.port, value)
+            value = self.process_bitwise_assign(action.port, name, value)
```

Comment:
Just confirming this will work for nested arrays? Like `tester.circuit.I[0][0] = 1` where `I` is `Array[Bits[2], 2]`? I think the line above will give us the flattened thing?

Reply:
Those should be flattened out by this point (since coreir flattens out arrays down to arrays of bits), but I'll add a test to be sure